ros/test/test_main.cpp

```diff
@@ -60,86 +61,87 @@ TEST(test_main, basic)
-
-    InertialSenseROS isROS(config);
-    isROS.initialize();
-
-    testNode.quiet = true;
-
-    bool success = false;
-    unsigned int startTimeMs = current_timeMs(), prevTimeMs = 0, nowTimeMs;
-    while( ((nowTimeMs = current_timeMs()) - startTimeMs < 10000) && !testNode.got_gps_tow )
-    {
-        isROS.update();
-        // check regularly, but don't print regularly..
-        SLEEP_MS(100);
-        if (prevTimeMs / 2500 != nowTimeMs / 2500) {
-            TEST_COUT << "Waiting for GPS Fix/TimeOfWeek Update...  (time: " << (nowTimeMs - startTimeMs) / 1000.0 << ")" << std::endl;
-            prevTimeMs = nowTimeMs;
-        }
-    }
-
-    ASSERT_TRUE(testNode.got_gps_tow) << "Timeout waiting for TimeOfWeek to set (Poor GPS Signal?). Unable to perform relevant tests." << std::endl;
-    TEST_COUT << "Got TimeOfWeek/GPS Fix.  Collecting 10 seconds of data..." << std::endl;
-
-    startTimeMs = current_timeMs(), prevTimeMs = 0;
-    while((nowTimeMs = current_timeMs()) - startTimeMs < 10000)
-    {
-        isROS.update();
-        // check regularly, but don't print regularly..
-        SLEEP_MS(100);
-        if (prevTimeMs / 2500 != nowTimeMs / 2500) {
-            TEST_COUT << "Collecting data...  (" << 10.0 - (nowTimeMs - startTimeMs) / 1000.0 << "sec remaining)" << std::endl;
-            prevTimeMs = nowTimeMs;
-        }
-    }
-
-    TEST_COUT << "Timestamp Deviation (GPS <> pIMU):  [" << testNode.get_min_deviation(testNode.gps_ts, testNode.pimu_ts) << " <= " <<  testNode.get_avg_deviation(testNode.gps_ts, testNode.pimu_ts) << " <= "  << testNode.get_max_deviation(testNode.gps_ts, testNode.pimu_ts) << "]" << :: std::endl;
-    TEST_COUT << "Timestamp Deviation (GPS <> IMU):   [" << testNode.get_min_deviation(testNode.gps_ts, testNode.imu_ts)  << " <= " <<  testNode.get_avg_deviation(testNode.gps_ts, testNode.imu_ts) << " <= "  << testNode.get_max_deviation(testNode.gps_ts, testNode.imu_ts) << "]" << :: std::endl;
-    TEST_COUT << "Timestamp Deviation (GPS <> INS):   [" << testNode.get_min_deviation(testNode.gps_ts, testNode.ins_ts)  << " <= " <<  testNode.get_avg_deviation(testNode.gps_ts, testNode.ins_ts) << " <= "  << testNode.get_max_deviation(testNode.gps_ts, testNode.ins_ts) << "]" << :: std::endl;
-    TEST_COUT << "Timestamp Deviation (INS <> IMU):   [" << testNode.get_min_deviation(testNode.ins_ts, testNode.imu_ts)  << " <= " <<  testNode.get_avg_deviation(testNode.ins_ts, testNode.imu_ts) << " <= "  << testNode.get_max_deviation(testNode.ins_ts, testNode.imu_ts) << "]" << :: std::endl;
-
-    EXPECT_GE( 0.05,  testNode.get_avg_deviation(testNode.gps_ts, testNode.pimu_ts));
-    EXPECT_GE( 0.05,  testNode.get_avg_deviation(testNode.gps_ts, testNode.imu_ts));
-    EXPECT_GE( 0.05,  testNode.get_avg_deviation(testNode.gps_ts, testNode.ins_ts));
-    EXPECT_GE( 0.005, testNode.get_avg_deviation(testNode.ins_ts, testNode.imu_ts));
-    isROS.terminate();
-}
+// TEST(test_main, gps_ins_time_sync)
```

Comment:
Can we put this in a "#if 0  ...  #endif" block instead of as a giant block of single-line comments?
Also, be sure to add a comment after the `#if 0` that explains why its been turned off.  Ie, "This test has been disabled for reason X..."